reaper.go

```diff
@@ -240,13 +240,18 @@ func newReaper(ctx context.Context, sessionID string, provider ReaperProvider) (
 		SessionID: sessionID,
 	}
 
-	listeningPort := nat.Port("8080/tcp")
+	listeningPort := nat.Port(config.ReaperDefaultPort)
 
 	tcConfig := provider.Config().Config
 
+	exposedPort := string(listeningPort)
+	if tcConfig.RyukAddress != "" {
+		exposedPort = net.JoinHostPort(tcConfig.RyukAddress, "") + ":" + exposedPort
```

Comment:
bug: pretty sure this would result in a double colon, I would expect the following, did I miss something?
```suggestion
	exposedPort := config.ReaperDefaultPort
	if tcConfig.RyukAddress != "" {
		exposedPort = net.JoinHostPort(tcConfig.RyukAddress, exposedPort)
```

Also note that the other PR has port configurable using the env var `RYUK_PORT`, not sure if that will land, but just making you aware.